k8s/upgrades/0.9.0-1.0.0/jiva/verify_volume_upgrade.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+target_upgrade_version=1.0.0
+is_upgrade_failed=0
+
+function usage() {
+    echo
+    echo "Usage:"
+    echo
+    echo "$0 <pv-name>"
+    echo
+    echo "  <pv-name> Get the PV name using: kubectl get pv"
+    exit 1
+}
+
+if [ "$#" -ne 1 ]; then
+    usage
+fi
+
+pv=$1
+
```

Comment:
We can print message saying `verifying jiva $pv volume upgrade`

Reply:
Done